_sass/scaffolds.scss

```diff
@@ -370,7 +386,11 @@ li.sticky-nav{
 
 @media (min-width: 1281px) {
   body, p, ol, ul, td, pre {
-    font-size: 22px;
+    font-size: 20px;
```

Comment:
should this be in this branch/PR? can it move to the other font changez branch?

Reply:
i can definitely move this, thanks for catching it.